control-plane/helper/test/test_util.go

```diff
@@ -282,7 +282,7 @@ func SetupK8sAuthMethodWithNamespaces(t *testing.T, consulClient *api.Client, se
 			Description: "Kubernetes binding rule",
 			AuthMethod:  AuthMethod,
 			BindType:    api.BindingRuleBindTypeTemplatedPolicy,
-			BindName:    "", //api.ACLTemplatedPolicyWorkloadIdentityName, TODO: remove w/ v2 code
+			BindName:    "builtin/workload-identity", // TODO: remove w/ v2 code
```

Comment:
Using a literal because in the corresponding `api` submodule version, this constant has been removed.